components/retro-go/rg_system.c

```diff
@@ -316,11 +316,11 @@ rg_app_t *rg_system_init(int sampleRate, const rg_handlers_t *handlers, const rg
     }
 
     // Now we init everything else!
+    rg_input_init(); // aw9523 goes first!
```

Comment:
Why must this be first? I can't think of any issue having it first but I'm just curious :) .

Reply:
because the tft reset line and backlight is connected through the gpio expander - we do a hard reset of the TFT to make sure its in a good state before initializing the display